connector_phone_palitto_event_manager/models/res_partner.py

```diff
@@ -47,23 +48,40 @@ def open_outgoing_notification(self):
     @api.multi
     def outgoing_call_notification(self):
         # For Outgoing Calls
+        #import pdb; pdb.set_trace()
         if self == {}:
             raise UserError(_("Bad Partner Record"))
         if not self.env.user.company_id.server_address:
             raise UserError(_("Please specify server address in Company Setting"))
-        server = self.env.user.company_id.server_address + "/DialNumber/?"
+        #server = self.env.user.company_id.server_address + "/DialNumber/?"
+        server = self.env.user.company_id.server_address
         number = self.called_for_mobile and self.mobile or self.phone  # Fetched from partner
 
         user = self.env.user
         ext = user.related_phone  # Fetched from user
+        #payload = {
+        #    "ext": ext,
+        #    "number": number,
+        #}
+        session_id = "60D94B51-176E-4866-ADD7-A9A9D8C30089"
+        endpoint = "/v3/cca/sessions/" + session_id + "/dial?Content-Type=application/json&token=AKXxaFOFkQJ5Qj72P1BWys7c3Z1wlammAqELqj9xVeA="
```

Comment:
@tirth-osi @b-kannan Why are we doing any change in Pallito module?  connector_phone_palitto_event_manager